packages/react-components/src/organisms/ResearchOutputIdentifier.tsx

```diff
@@ -65,6 +67,35 @@ const identifierMap = {
   },
 } as const;
 
+const getIdentifierInfoMessage = (
+  identifiers: IdentifierType,
+): Array<ReactElement> =>
+  identifiers
+    .filter(({ value }) => value !== ResearchOutputIdentifierType.None)
+    .map(({ value }) => {
+      if (value === ResearchOutputIdentifierType.AccessionNumber) {
+        return (
+          <span key={value}>
+            <b>Accesion Number: </b>Your Accession Number must start with a
+            letter. Accession Numbers are attributed by NIH, EMBL-EBI,
+            ProteomeXchange, etc.
+          </span>
```

Comment:
Should this be a component where we pass in the value, name and description? 

It's used in 3 places?